test/unit/active_model/serilizable_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+module ActiveModel
+  class SerializableTest
+    class InstrumentationTest < Minitest::Test
+      def setup
+        @events = []
+
+        ActiveSupport::Notifications.subscribe("!serialize.active_model_serializers") do |name, start, finish, id, payload|
+          @events << { name: name, serializer: payload[:serializer] }
+        end
+      end
+
+      def teardown
+        ActiveSupport::Notifications.unsubscribe("!serialize.active_model_serializers")
+      end
+
+      def test_default_serializer
```

Comment:
maybe `test_instruments_default_serializer` here and below, I think would better description of the test